components/logins/android/build.gradle

```diff
@@ -57,6 +50,7 @@ android {
     tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
         kotlinOptions {
             jvmTarget = rootProject.ext.build.jvmTargetCompatibility
+            freeCompilerArgs += "-Xopt-in=kotlin.ExperimentalUnsignedTypes"
```

Comment:
I don't think we have a Formal Policy on this, but in general I think we try not to opt in to experimental kotlin features at the library level, instead leaving that choice up to the application. I guess you were running into something like https://github.com/mozilla/application-services/issues/3957 here?

I think the plan outlined in https://github.com/mozilla/application-services/issues/3957#issuecomment-803169111 would be a tractable way to solve this in uniffi and avoid the need for us to specify the opt-in here. @skhamis if I wrote it up in a bit more detail, I wonder if you'd be interested in having a go at implementing the fix in uniffi?

Reply:
I definitely would! The only reason I had it do it in the build.gradle was I couldn't get the @Expiremental... annotations in the generated files. Also good that it lets the consumer know that kotlin deems it experimental

Reply:
Ref https://github.com/mozilla/uniffi-rs/issues/444 for the corresponding UniFFI bug.